Makefile.am

```diff
@@ -6,7 +6,7 @@ SUBDIRS += rpm
 endif
 
 if CONFIG_USER
-SUBDIRS += etc man scripts lib tests cmd contrib
+SUBDIRS += man scripts lib tests cmd etc contrib
```

Comment:
Is this ordering change significant?

Reply:
https://github.com/openzfs/zfs/runs/2384354268?check_suite_focus=true#step:6:1299